frontend/ui/src/app/tutorials/branding/branding-tutorial.component.html

```diff
@@ -0,0 +1,216 @@
+                </p>
+                <p class="mt-8">
+                  You can find additional resources on this topic in our
+                  <a
+                    href="https://distr.sh/docs/product/customer-portal/"
+                    class="text-primary-600 underline"
+                    target="_blank"
+                    >docs</a
+                  >.
+                </p>
+              </div>
+
+              <div class="gap-4 sm:flex sm:items-center sm:justify-between">
+                <a
+                  [routerLink]="['..']"
+                  class="flex-1 sm:flex-none items-center justify-center w-full rounded-lg border border-gray-200 bg-white px-5 py-2.5 text-sm font-medium text-gray-900 hover:bg-gray-100 hover:text-primary-700 focus:z-10 focus:outline-none focus:ring-4 focus:ring-gray-100 dark:border-gray-600 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white dark:focus:ring-gray-700 sm:w-auto">
+                  Exit Tutorial
+                </a>
+                <button
+                  (click)="continueFromWelcome()"
+                  type="submit"
+                  class="flex w-full items-center justify-center rounded-lg border border-primary-700 bg-primary-700 px-5 py-2.5 text-sm font-medium text-white hover:border-primary-800 hover:bg-primary-800 focus:outline-none focus:ring-4 focus:ring-primary-300 dark:border-primary-700 dark:bg-primary-600 dark:hover:border-primary-700 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:mt-0 sm:w-auto">
+                  Let's Go
+                  <fa-icon class="ml-2" [icon]="faArrowRight"></fa-icon>
+                </button>
+              </div>
+            </div>
+          </cdk-step>
+          <cdk-step label="Branding Options" [stepControl]="brandingFormGroup">
+            <div class="p-8 w-full">
+              <form [formGroup]="brandingFormGroup" (ngSubmit)="continueFromBranding()">
+                <ol
+                  class="ml-4 relative text-gray-900 dark:text-white mb-12 border-s border-gray-200 dark:border-gray-700">
+                  <li class="mb-10 ms-8">
+                    <span
+                      class="absolute flex items-center justify-center w-8 h-8 bg-gray-100 rounded-full -start-4 ring-4 ring-white dark:ring-gray-900 dark:bg-gray-700">
+                      <fa-icon [icon]="faLightbulb"></fa-icon>
+                    </span>
+                    <h3 class="font-medium leading-tight">Branding Options</h3>
+                    <p class="text-sm mt-2">
+                      Imagine you are one of your customers: What would you like to see in the Distr Customer Portal?
+                    </p>
+                  </li>
+                  <li class="mb-10 ms-8">
+                    <span class="absolute flex items-center justify-center -start-2">
+                      <input type="checkbox" [formControl]="brandingFormGroup.controls.titleDone" />
```

Comment:
Checkboxes look different than the other checkboxes in our platform: missing round corners (e.g. Show Archived Versions)

![image](https://github.com/user-attachments/assets/1e89bac5-3c58-40cb-8404-c202525346fb)